src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -157,78 +157,77 @@ bool Deconvolution::isSupportedOperation(const std::shared_ptr<const ngraph::Nod
 Deconvolution::Deconvolution(const std::shared_ptr<ngraph::Node>& op,
                              const GraphContext::CPtr context) : Node(op, context, DeconfolutionShapeInferFactory(op)) {
     std::string errorMessage;
-    if (isSupportedOperation(op, errorMessage)) {
-        errorPrefix = "Deconvolution node with name '" + getName() + "'";
 
-        auto convBackprop = std::dynamic_pointer_cast<const ngraph::opset1::ConvolutionBackpropData>(op);
-        auto groupConvBackprop = std::dynamic_pointer_cast<const ngraph::opset1::GroupConvolutionBackpropData>(op);
-        const auto& weightDims = getWeightDims();
+    if (!isSupportedOperation(op, errorMessage))
+        IE_THROW(NotImplemented) << errorMessage;
```

Comment:
Just wondering, should not we augment the error message with the error prefix to have more info on hand when the error occurs?